request/request.js

```diff
@@ -155,6 +155,13 @@ module.exports = function($window, Promise, oncompletion) {
 				}
 			}
 
+			xhr.ontimeout = function (ev) {
+				if (aborted) return;
+				var error = new Error("Request timeout");
```

Comment:
Let's be a little clearer on the error message

```suggestion
				var error = new Error("Request timed timeout");
```